requirements.txt

```diff
@@ -3,7 +3,7 @@ altair>=4.2.0,<6.0
 fastapi
 ffmpy
 gradio_client==0.10.0
-httpx
+httpx>=0.24.1
```

Comment:
Let's do so for the python client as well?

Reply:
This comment led me down a deep rabbit hole. But the upside is that I know our CI a lot better now. Thanks @freddyaboulton! I've addressed it so I'll go ahead and merge